src/main/java/uk/gov/gchq/palisade/rule/WrappedRule.java

```diff
@@ -124,7 +124,11 @@ public T apply(final T obj, final User user, final Context context) {
             rtn = function.apply(obj);
         } else if (nonNull(predicate)) {
             final boolean test = predicate.test(obj);
-            rtn = test ? obj : null;
```

Comment:
I'd say the ternary operator is easier to read in cases like this

Reply:
I would still say the more verbose if statements are easier to read. Especially given anyone who can code understands if statements. Not everyone understands the ternary notation